eos_data_distribution/producers/usb.py

```diff
@@ -107,7 +107,6 @@ def main():
     GLib.unix_signal_add(GLib.PRIORITY_DEFAULT, signal.SIGTERM, signal_cb)
     monitor = Gio.VolumeMonitor.get()
     store = SimpleStoreProducer(prefix=SUBSCRIPTIONS_SOMA,
-                                split=defaults.ENDLESS_NDN_CACHE_PATH,
```

Comment:
i cherry-picked your dbus_service fix in my performance branch, (that you validated) there is another fix for base crash too. maybe we can drop that branch and merge the other ?